torch/onnx/_internal/exporter/_ir_passes.py

```diff
@@ -39,3 +42,29 @@ def add_torchlib_common_imports(model: ir.Model) -> None:
         model.functions[is_scalar_func.identifier()] = is_scalar_func
     except Exception:
         logger.exception("Failed to add torchlib common imports to the model.")
+
+
+def _get_opset_version(node: ir.Node, opset_imports: Mapping[str, int]) -> int:
+    """Determine the appropriate opset version for a node."""
+    domain = node.domain
+    version = node.version if node.version is not None else 1
+    if domain == "":
+        return max(version, _MIN_ONNX_OPSET_VERSION)
```

Comment:
Why do we need that? Either the exporter does not support opset < 18 and this never happens, either it does and this should not be needed.

Reply:
actually we do need this line because I use it to collect the opset version for the model. When the model has all ops in <18 we still need to set the model version to 18. This line ensures we always get a version >18. Otherwise the model version will be <18.